pop.css

```diff
@@ -0,0 +1,148 @@
+.projectPopup {
```

Comment:
- [ ] I can see a few differences between your design and Figma design,  please note that for your pop-up windows that it is very important to stick to the Figma design as much as possible in order to have a beautiful website and good user experience 👍. Your popup and image height can be improved, and your button also, and elements spacing(padding/margin) line height and position are a bit different, please check the screenshots below ⬇️ to see the differences.  Please make sure that your mobile menu design is similar to the figma template as required for this project 👍

#### Figma design
![modal](https://user-images.githubusercontent.com/6615394/223975029-e4037af8-f63f-48d3-af98-d904e1bc64e4.jpg)

####  Your design

![Capture](https://user-images.githubusercontent.com/6615394/223975834-cdeeefa1-5a8b-4ec6-a6ab-7fadc8beca9e.PNG)